app/controllers/application_controller.rb

```diff
@@ -9,20 +9,26 @@ class ApplicationController < ActionController::Base
   before_filter :update_last_known_latlng
 
   helper_method :bugsnag_api_key
+  helper_method :bugsnag_api_key?
   helper_method :google_universal_analytics_tracking_code
+  helper_method :google_universal_analytics_tracking_code?
 
   private
 
   def bugsnag_api_key
-    if Rails.env.production?
-      ENV['BUGSNAG_API_KEY']
-    end
+    ENV['BUGSNAG_API_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.